pbrd/pbr_vty.c

```diff
@@ -899,17 +980,30 @@ static void vty_show_pbrms(struct vty *vty,
 		vty_out(vty, "        SRC Port Match: %u\n", pbrms->src_prt);
 	if (pbrms->dst_prt)
 		vty_out(vty, "        DST Port Match: %u\n", pbrms->dst_prt);
+
 	if (pbrms->dsfield & PBR_DSFIELD_DSCP)
-		vty_out(vty, "        DSCP Match: %u\n",
+		vty_out(vty, "        Match DSCP %u\n",
```

Comment:
please put this back someone might be screen scraping this right?

Reply:
restored to original